src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -111,10 +110,8 @@ public Routes(
      */
     @Override
     public CompletionStage<HttpResponse> apply(final HttpRequest request) {
-        final Metrics metrics = _metricsFactory.create();
-        final Timer timer = metrics.createTimer(createMetricName(request, REQUEST_METRIC));
-        final Counter bodySize = metrics.createCounter(createMetricName(request, BODY_SIZE_METRIC));
-        bodySize.increment(request.entity().getContentLengthOption().orElse(0L));
+        final Stopwatch requestTimer = Stopwatch.createStarted();
```

Comment:
Would like to hear about StopWatch versus our Timer and whether we should have a StopWatch if this is the pattern.